php/src/Container/ContainerState.php

```diff
@@ -2,11 +2,35 @@
 
 namespace AIO\Container;
 
-enum ContainerState: string {
-    case ImageDoesNotExist = 'image_does_not_exist';
-    case NotRestarting = 'not_restarting';
-    case Restarting = 'restarting';
-    case Running = 'running';
-    case Starting = 'starting';
-    case Stopped = 'stopped';
+enum ContainerState {
+    case DoesNotExist;
+    case Restarting;
+    case Healthy;
+    case Starting;
+    case Stopped;
+    case Unhealthy;
+
+    public function isStopped(): bool {
+        return $this == self::Stopped;
+    }
+
+    public function isStarting(): bool {
+        return $this == self::Starting;
+    }
+
+    public function isRestarting(): bool {
+        return $this == self::Restarting;
+    }
+
+    public function isHealthy(): bool {
+        return $this == self::Healthy;
+    }
+
+    public function isUnhealthy(): bool {
+        return $this == self::Unhealthy;
+    }
+
+    public function isRunning(): bool {
+        return $this->isHealthy() || $this->isUnhealthy() || $this->isStarting() || $this->isRestarting();
+    }
```

Comment:
The Running state corresponds to the old GetRunningContainerState state.
Maybe there is another clearer state. Healthy means that the container is running without any problem detected by the AIO.